libs/Microsoft.MixedReality.WebRTC.Native/include/interop_api.h

```diff
@@ -209,26 +290,66 @@ enum class TrackKind : uint32_t {
   kDataTrack = 3,
 };
 
-/// Callback fired when a remote track is added to a connection.
-using PeerConnectionTrackAddedCallback = void(MRS_CALL*)(void* user_data,
-                                                         TrackKind track_kind);
-
-/// Callback fired when a remote track is removed from a connection.
-using PeerConnectionTrackRemovedCallback =
-    void(MRS_CALL*)(void* user_data, TrackKind track_kind);
+/// Callback fired when a remote audio track is added to a connection.
+/// The |audio_track| and |audio_transceiver| handle hold a reference to the
+/// underlying native object they are associated with, and therefore must be
+/// released with |mrsLocalAudioTrackRemoveRef()| and
+/// |mrsAudioTransceiverRemoveRef()|, respectively, to avoid memory leaks.
+using PeerConnectionAudioTrackAddedCallback =
```

Comment:
These should have a prefix too I suppose?

Reply:
Fixed.